mava/wrappers/observation.py

```diff
@@ -34,28 +36,19 @@ def __init__(self, env: Environment, has_global_state: bool = False):
     def _add_agent_ids(
         self, timestep: TimeStep, num_agents: int
     ) -> Union[Observation, ObservationGlobalState]:
+        """Adds agent IDs to the observation."""
```

Comment:
Double check this, but I *think* we don't need the `has_global_state` arg anymore?

Reply:
Yes with the .replace() we can remove the `has_global_state` flag.